wqflask/wqflask/metadata_edits.py

```diff
@@ -171,7 +179,7 @@ def update_phenotype(dataset_id: str, name: str):
         or ""
     )
     phenotype_id = str(data_.get("phenotype-id"))
-    if not (file_ := request.files.get("file")):
+    if not (file_ := request.files.get("file")) and data_.get('edited') == "false":
```

Comment:
*question: issue:* This seems odd to me.  Doing a string comparison the string "false".  If there's a way to do directly with a boolean value, it would make this easier to both debug and read.